opm/parser/eclipse/Units/UnitSystem.cpp

```diff
@@ -129,6 +138,9 @@ namespace {
         1, /* gas-oil ratio */
```

Comment:
Are there many ratios in inputs? I'd assume not, but regardless I think input is mostly handled by the string-based lookup which hasn't been changed at all.